Packs/InfobloxBloxOne/Integrations/InfobloxBloxOneThreatDefenseEventCollector/InfobloxBloxOneThreatDefenseEventCollector.py

```diff
@@ -0,0 +1,116 @@
+from CommonServerPython import *
+from CommonServerUserPython import *
+import demistomock as demisto
+
+
+VENDOR = 'Infoblox BloxOne'
+PRODUCT = 'Threat Defense'
+
+
+class BloxOneTDEventCollectorClient(BaseClient):
+    def __init__(self, api_key: str, verify=True, proxy=False):
+        super(BloxOneTDEventCollectorClient, self).__init__(
+            headers={'Authorization': f'Token {api_key}'},
+            base_url='https://csp.infoblox.com',
+            verify=verify,
+            proxy=proxy
+        )
+
+    def fetch_events(self, from_ts: int, to_ts: int, limit: int = 1000, offset: int = 0) -> list[dict]:
+        def map_time(event: dict) -> dict:
+            event['_time'] = event.get('event_time')
+            return event
+
+        events = self._http_request('GET', '/api/dnsdata/v2/dns_event',
+                                    params={'t0': from_ts, 't1': to_ts, '_limit': limit, '_offset': offset}
+                                    ).get('result', [])
+        return list(map(map_time, events))
+
+
+def fetch_events_command(client: BloxOneTDEventCollectorClient, params: dict, last_run: dict):
+    from_ts = last_run.get('from_ts') or parse_from_ts_from_params(params.get('first_fetch'))
+    current_ts = int(datetime.utcnow().timestamp())
+    offset = arg_to_number(last_run.get('offset')) or 0
+    limit = arg_to_number(params.get('max_fetch')) or 1000
+    limit = min(limit, 10000)
+    events = client.fetch_events(from_ts, current_ts, limit, offset)
+
+    send_events_to_xsiam(events, VENDOR, PRODUCT)
+    demisto.setLastRun(
+        {'from_ts': current_ts} if len(events) < limit else {'from_ts': from_ts, 'offset': offset + limit}
+    )
+
+
+def get_events_command(client: BloxOneTDEventCollectorClient, args: dict):
+    events = client.fetch_events(
+        args['from'], args['to'],
+        min(arg_to_number(args.get('limit')) or 1000, 10000),
+        arg_to_number(args.get('offset')) or 0
+    )
+    if argToBoolean(args.get('should_push_events', False)):
+        send_events_to_xsiam(events, VENDOR, PRODUCT)
+
+    return CommandResults(outputs=events, outputs_prefix="test_get_events")
```

Comment:
`outputs_prefix="test_get_events"` is not good. Please change it to someone that matches our standards, and document it in the readme.